toolchain/check/import.cpp

```diff
@@ -617,4 +541,68 @@ auto ImportNameFromOtherPackage(
   return result_id;
 }
 
+auto AddImportNamespace(Context& context, SemIR::TypeId namespace_type_id,
```

Comment:
I can't view a delta because you moved the code. Can you please clarify why this needed to be moved, or to help me in review (and maintain history), move it back?

Reply:
I've moved it to match the order of the methods in the .h.
I'm happy to try and do this move in a separate PR or move the entire refactoring to another PR if you think it's better.
It might make sense to move it to a completely separate .h/.cpp.
Eventually the code is likely to be moved unless we're preferring blame history over code style.
Let me know what you prefer.

Reply:
> I've moved it to match the order of the methods in the .h.

How about doing this the other way, putting it in the .h in a spot that matches the order in the cpp file? i.e., don't put it last, put it first. Then I think we can get the best of both worlds: order in both would match, and it wouldn't need to move - in this PR or any other.

Reply:
Done.
However, I feel this should not be the final location.
1. IMO, iIt makes more sense to put this function declared down because it's not the primary `import.h` API (and perhaps should actually move to a different file). BTW, when moving code to a different file, can we make it easy to review or keep blame history? Do we usually do that? Do we avoid moving code between files because of that?
2. The definition is now between internal linkage functions instead of being close to the API functions. I don't think this is explicitly against the style guide, but definitely seems inconsistent.

Should I move the code to a better location (after you review or in a separate PR)? If so, how do you feel about moving it to a different file so we don't need to include `import.h` just for that?